app/views/internal/broadcasts/_form.html.erb

```diff
@@ -14,3 +14,12 @@
   <%= select_tag :active, options_for_select([false, true]) %>
 </div>
 <br>
+
+<% if @broadcast.processed_html %>
+  <div>
+    </p><strong>Preview</strong></p>
+    <%= @broadcast.processed_html.html_safe %>
```

Comment:
Hm, looks like brakeman is complaining that this is an `Unescaped model attribute`? This is a new one for me, but from the source code it looks like brake man [does indeed think](https://github.com/presidentbeef/brakeman/blob/c9fc366db717f759c064371a0321cccc88c2affb/test/apps/rails4/app/views/another/html_safe_is_not.html.erb) that this is unsafe.

Not sure what is the best way to get around this, since we _do_ want to be able to let admins preview here, and this html is a value that comes _directly from the database_ and not a cookie/param. Any thoughts? 🤔

Reply:
Huh... 🤔 Maybe something like this could be useful?: https://stackoverflow.com/questions/39099489/how-to-fix-cross-site-scripting-security-warning-in-rails-when-using-html-safe

Reply:
@juliannatetreault Great find!! Trying it out with `sanitize`.

Sadly, it seems I need to explicitly pass in the sanitized attributes (according to [the docs](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize_css) `style` and css should be sanitized by default), but I think that might be because of the way sanitize works with nested elements. In any case, I think passing in explicitly is safer.

Thanks again sending that method my way! 👍